src/ListCatalog.js

```diff
@@ -70,27 +72,79 @@ class ListItem extends Component {
     }
   }
 
+  renderTrailingCheckbox() {
+    this.initCheckbox = checkboxEl => checkboxEl && this.destroyableComponents.push(new MDCCheckbox(checkboxEl));
+
+    if (this.props.trailingCheckbox) {
+      return (
+          <div className='mdc-checkbox mdc-list-item__meta' ref={this.initCheckbox}>
+            <input type='checkbox'
+                   defaultChecked={true}
+                   className='mdc-checkbox__native-control'/>
+            <div className='mdc-checkbox__background'>
+              <svg className='mdc-checkbox__checkmark'
+                   viewBox='0 0 24 24'>
+                <path className='mdc-checkbox__checkmark-path'
+                      fill='none'
+                      stroke='white'
+                      d='M1.73,12.91 8.1,19.28 22.79,4.59'/>
+              </svg>
+              <div className='mdc-checkbox__mixedmark'></div>
+            </div>
+          </div>
+      )
+    }
+  }
+
+  renderTrailingRadioButton() {
+    this.initRadio = radioEl => radioEl && this.destroyableComponents.push(new MDCRadio(radioEl));
+
+    if (this.props.trailingRadio) {
+      return (
+          <div className='mdc-radio mdc-list-item__meta' ref={this.initRadio}>
+            <input className='mdc-radio__native-control'
+                   type='radio'
+                   name='listDemoRadioGroup'
+                   defaultChecked='false' />
+            <div className='mdc-radio__background'>
+              <div className='mdc-radio__outer-circle'/>
+              <div className='mdc-radio__inner-circle'/>
+            </div>
+          </div>
+      )
+    }
+  }
+
   render() {
     const {className, activated} = this.props;
     const classes = classnames('mdc-list-item', className, {'mdc-list-item--activated': activated});
 
     return (
-      <li className={classes} ref={this.initRipple}>
+      <li className={classes} ref={this.initRipple} tabIndex='-1'>
```

Comment:
I was not able to tab through the checkbox & radio list groups. We might need to set tabIndex to `0` for first list item.